Earthfile

```
@@ -0,0 +1,25 @@
+VERSION 0.6
+FROM alpine:3.14
+WORKDIR /src
+
+build:
+    FROM rust:alpine3.14
+    RUN apk update \
+        && apk add \
+            git \
+            gcc \
+            g++ \
+            pkgconfig
+
+    COPY . /src
+    WORKDIR /src
+    RUN cargo build --release
+    SAVE ARTIFACT /src/target/release/tokei AS LOCAL ./tokei
+
+docker:
+    BUILD +build
+    COPY +build/tokei /usr/local/bin/
```

I may be wrong, but I believe you can remove the `BUILD` instruction here because referencing the `+build/tokei` artifact causes it to build.
```suggestion
    COPY +build/tokei /usr/local/bin/
```